.sonarcloud.properties

```diff
@@ -1,3 +1,2 @@
 sonar.sources=./Sources
-sonar.coverageReportPaths=./artifacts/coverage/sonarqube-generic-coverage.xml
```

Comment:
Sonarcloud Github PR check doesn't support code coverage reports

Reply:
We're using the Sonarcloud GitHub app for this? Maybe a GitHub Action (with sonarcloud-github-action) would work?

Reply:
Not sure. That's expected behaviour for PR check. We send coverage data to SonarQube - that should be enough for now